[2025-05-25T05:27:29.498+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-05-25T05:27:29.938+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: load_file_to_psql.move_data_file scheduled__2025-02-02T00:00:00+00:00 [queued]>
[2025-05-25T05:27:30.360+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: load_file_to_psql.move_data_file scheduled__2025-02-02T00:00:00+00:00 [queued]>
[2025-05-25T05:27:30.377+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-05-25T05:27:30.720+0000] {taskinstance.py:2889} INFO - Executing <Task(BashOperator): move_data_file> on 2025-02-02 00:00:00+00:00
[2025-05-25T05:27:30.795+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=879) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-05-25T05:27:30.839+0000] {standard_task_runner.py:72} INFO - Started process 923 to run task
[2025-05-25T05:27:30.876+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'load_file_to_psql', 'move_data_file', 'scheduled__2025-02-02T00:00:00+00:00', '--job-id', '118', '--raw', '--subdir', 'DAGS_FOLDER/data_quality.py', '--cfg-path', '/tmp/tmp2bhxizu5']
[2025-05-25T05:27:31.043+0000] {standard_task_runner.py:105} INFO - Job 118: Subtask move_data_file
[2025-05-25T05:27:36.181+0000] {task_command.py:467} INFO - Running <TaskInstance: load_file_to_psql.move_data_file scheduled__2025-02-02T00:00:00+00:00 [running]> on host b4cf172ed0b9
[2025-05-25T05:27:39.045+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='https://t.me/Shust_DE' AIRFLOW_CTX_DAG_OWNER='@Shust_DE' AIRFLOW_CTX_DAG_ID='load_file_to_psql' AIRFLOW_CTX_TASK_ID='move_data_file' AIRFLOW_CTX_EXECUTION_DATE='2025-02-02T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-02-02T00:00:00+00:00'
[2025-05-25T05:27:39.078+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-05-25T05:27:40.190+0000] {subprocess.py:78} INFO - Tmp dir root location: /tmp
[2025-05-25T05:27:40.333+0000] {subprocess.py:88} INFO - Running command: ['/usr/bin/bash', '-c', 'mkdir -p /tmp/processed_data/ && mv /tmp/data.csv /tmp/processed_data/']
[2025-05-25T05:27:40.996+0000] {subprocess.py:99} INFO - Output:
[2025-05-25T05:27:41.243+0000] {subprocess.py:110} INFO - Command exited with return code 0
[2025-05-25T05:27:41.690+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-05-25T05:27:41.694+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=load_file_to_psql, task_id=move_data_file, run_id=scheduled__2025-02-02T00:00:00+00:00, execution_date=20250202T000000, start_date=20250525T052730, end_date=20250525T052741
[2025-05-25T05:27:42.430+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-05-25T05:27:42.564+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
